### ISSUE TITLE
EL9 support: change PID file location to /run

### ISSUE THREAD

**Issue by jouvin** [quattor/cdp-listend#30]
On EL9, you get a warning if your systemd configuration refers to `/var/run` instead of `/run`:

```
Mar 19 08:33:00 psonar1 systemd[1]: /usr/lib/systemd/system/cdp-listend.service:11: PIDFile= references a path below legacy directory /var/run/, updating /var/run/cdp-listend.pid → /run/cdp-listend.pid; please update the unit file accordingly.
```

**Comment by stdweird** [comment]
if this is ok on el8 as well, you can change it on https://github.com/quattor/cdp-listend/blob/master/src/main/resources/systemd/cdp-listend.service#L11

**Comment by jouvin** [comment]
It is ok since EL7... (where `/var/run` is a link to `/run`)

**Comment by jrha** [comment]
Fixed in #33.
